lib/Conversion/MooreToCore/MooreToCore.cpp

```diff
@@ -104,27 +105,31 @@ static hw::ModulePortInfo getModulePortInfo(const TypeConverter &typeConverter,
 }
 
 /// Erase all dead blocks in a region.
-static void eraseDeadBlocks(PatternRewriter &rewriter, Region &region) {
+static void eraseDeadBlocks(Region &region) {
```

Comment:
Can we use [mlir::eraseUnreachableBlocks](https://mlir.llvm.org/doxygen/namespacemlir.html#a6ab9585c8607c12142232e0f58ebcc8b)?